clar2wasm/Cargo.toml

```diff
@@ -5,15 +5,14 @@ edition = "2021"
 build = "build.rs"
 
 [dependencies]
-clarity = { git = "https://github.com/stacks-network/stacks-core.git", branch = "feat/clarity-wasm-develop", features = [
-  "testing",
-] }
 clap = { version = "4.3.17", features = ["derive"] }
 regex = "1.9.1"
 walrus = "0.20.1"
 lazy_static = "1.4.0"
 wasmtime = "15.0.0"
-stacks-common = { git = "https://github.com/stacks-network/stacks-core.git", branch = "feat/clarity-wasm-develop" }
+
+clarity = { git="https://github.com/stacks-network/stacks-core", branch="refactor/use-relative-path-for-workspace-dependencies", features = ["testing"] }
+stacks-common = { git="https://github.com/stacks-network/stacks-core", branch="refactor/use-relative-path-for-workspace-dependencies" }
 
```

Comment:
Are you going to change that to `feat/clarity-wasm-develop` ? The other PR in stacks-core is already approved.